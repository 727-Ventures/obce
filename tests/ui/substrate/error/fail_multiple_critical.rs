#[obce::error]
enum Error {
    One(u32),
    #[obce(critical)]
    Two(CriticalError),
    #[obce(critical)]
    Three(CriticalError)
}

fn main() {}
