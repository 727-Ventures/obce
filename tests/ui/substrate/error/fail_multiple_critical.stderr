error: only one enum variant can be marked as `#[obce(critical)]`
 --> tests/ui/substrate/error/fail_multiple_critical.rs:1:1
  |
1 | #[obce::error]
  | ^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `obce::error` (in Nightly builds, run with -Z macro-backtrace for more info)
